irodsadmin/tests/test_commands.py

```diff
@@ -471,4 +497,34 @@ def test_command_multiple(self):
             self.project.full_title,
             orphan_path,
         )
-        self.assertEqual(expected, out.getvalue())
+        self.assertEqual(expected, output)
+
+    def test_command_ordering(self):
+        """Test ordering of orphans in command output"""
+        project1 = self.make_project('A_Project', PROJECT_TYPE_PROJECT, None)
+        project2 = self.make_project('B_Project', PROJECT_TYPE_PROJECT, None)
```

Comment:
This tests looks good, but one small thing: Since we already create `self.project` in `setUp()`, we only need to create one addlitional project here within the test case. Otherwise the original project is needlessly used and we waste many precicious CPU cycles ;)